test/zdtm.py

```diff
@@ -2030,7 +2031,7 @@ def run_test(self, name, desc, flavor):
             "start": time.time()
         }
 
-        if test_flag(desc, 'excl'):
+        if test_flag(desc, 'excl') or link_remap_excl:
```

Comment:
If such tests will create a temporary directories and create their files there, will it solve the problem?

Reply:
Yes.